src/Popup/utils.js

```diff
@@ -138,7 +141,10 @@ const wmsSelector = (layer, event) => {
     if (features.length) {
       resolve({ features, layer })
     } else {
-      resolve({ features: [getPixelValue(layer, event)], layer })
+      const pixelValue = getPixelValue(layer, event)
+      const pixelFeatures = !!pixelValue ? [pixelValue] : []
```

Comment:
Won't this get wonky if the `pixelValue === 0` (e.g. falsey)

Reply:
@IDisposable yea totally considered this- meant to check on the return type from `getPixelValue` ...lack of typescript strikes again 😩